src/components/profile/ProfileTeamLocation.vue

```diff
@@ -71,13 +87,58 @@ export default {
     this.interval = window.setInterval(() => {
       this.localtime = this.getLocaltime();
     }, 1000);
+    this.getCurrentLocation();
   },
   beforeDestroy() {
     if (this.interval) {
       window.clearInterval(this.interval);
     }
   },
   methods: {
+    processLocation(position) {
+      const timezone = geoTz(
+        position.coords.latitude,
+        position.coords.longitude,
+      );
+      if (timezone.length === 0) {
+        return;
+      }
+      this.currentTimezone = timezone[0];
+    },
+    processLocationError(error) {
+      console.log('Unable to retrieve location error: ', error.message);
+      this.currentTimezone = this.$store.state.user.timezone.value;
```

Comment:
I think we should fallback to the system local time. That's most likely accurate.